File: Android/Java/Asg2 - Ice Cream App/app/src/main/java/com/example/asg2icecreamapp/OrderItem.java
package com.example.asg2icecreamapp;

import java.io.Serializable;
import java.util.Date;

public class OrderItem implements Serializable {
    Date date;
    String flavor;
    String size;
    String cost;

    OrderItem(String flavor, String size, String cost) {
        date = new Date();
        this.flavor = flavor;
        this.size = size;
        this.cost = cost;
    }

    @Override
    public String toString() {
        return  "Date: " + date + '\n' +
                "Flavor: " + flavor + '\n' +
                "Size: " + size + '\n' +
                "Cost: " + cost;
    }
}

File: Android/Kotlin/Asg3 Ice Cream App/app/src/main/java/com/example/asg3icecreamapp/OrderItem.kt
package com.example.asg3icecreamapp

import java.io.Serializable
import java.util.Date

class OrderItem (private val flavor:String, private val size:String, private val cost:String): Serializable {
    private val date = Date()
    override fun toString(): String {
        return  "Date: $date\n" +
                "Flavor: $flavor\n" +
                "Size: $size\n" +
                "Cost: $cost"
    }
}